scripts/upload-images-to-s3/uploadImagesToS3.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const logger = require('../../server/util/logger');
+const Position = require('../../server/models/Position');
+require('dotenv').config({ path: '../../.env' });
+
+const bucketName = process.env.AWS_S3_BUCKET_NAME;
+const accessKey = process.env.AWS_ACCESS_KEY;
+const secretKey = process.env.AWS_SECRET_KEY;
+
+AWS.config.update({
+  accessKeyId: accessKey,
+  secretAccessKey: secretKey,
+});
+
+const s3 = new AWS.S3();
+
+const connection = process.env.MONGODB_CONNECTION;
+
+const upload = () => {
+  Position.find({ previewImageLink: { $exists: false } }).lean().then((positions) => {
+    positions.forEach(async (position) => {
+      const key = position.variant ?? position.baseOpening.name;
+      logger.info(key);
+      const path = `png/${key.replace('/', '-')}.png`;
+      const buf = position.previewImage.data;
+
+      await fs.writeFile(path, buf.toString(), 'base64', (err) => {
```

Comment:
I tried to use the data buffer directly from mongo to upload it to S3 but had trouble figuring it out. I just ended up writing the file and reading the stream.